cli/pkg/cmd/cluster/create.go

```diff
@@ -0,0 +1,136 @@
+package cluster
+
+import (
+	"context"
```

Comment:
can we run `goimports` against this file. seems the import sequence looks weird.